src/transformers/models/tvlt/feature_extraction_tvlt.py

```diff
@@ -30,7 +30,7 @@
     import torchaudio
 
     
-# Copied from WhisperFeatureExtractor
+# Copied from whisper
```

Comment:
You cannot use Copied from as it will mess up the "copied from" mechanism of the library.

In HuggingFace Transformers, one can add Copied from statements above classes/methods and then run make fix-copies from the root of the repo, which will ensure that a certain portion of code is exactly copied.

e.g. if you place Copied from transformers.models.whisper.feature_extraction_whisper.WhisperFeatureExtractor with Whisper->TVLT above this class and then run make fix-copies, it will copy all the code of Whisper for TVLT (not sure this is what you want).

If you just want to add a comment stating that this is inspired from Whisper, you can use "Adapted from" or "Inspired by" (as long as you don't use Copied from)

 one can use make fix-copies to